efsity/README.md

```diff
@@ -172,6 +172,24 @@ $ fct validateFileStructure -i ~/Workspace/fhir-resources/<project> -s ~/path/to
 If the file structure matches the schema then a positive result is printed to the terminal, otherwise an error 
 is thrown showing the issue.
 
+### Validating StructureMap 
+The tool supports validation of a single structureMap and questionnaire **(single mode)** and validation of all the project files for the questionnaires and structureMaps **(project mode)**. The tool maps the different questionnaires to the specified structureMaps and then generates the questionnaireResponses. Once the questionnaireResponses have been generated, the bundle is then generated with the respective resources ie.(= Patient, Condition). To do this run the following command:
+
+**Options**
+```
+-i or --input - path to project folder which needs to be validated
```

Comment:
I think on single mode since we know the input is always one questionnaire we should add a new optional flag -q for questionnaire. that would make -i optional as well

Reply:
or we just add to the documentation here that points to it. i.e. -i taking both a folder and a file

Reply:
Additionally i suppose the input folder should be the _questionnaire_ folder instead of the the whole project folder. 

Once we enforce a standard structure then we then don't need other flags just the one input flag that points to the project